staker-rewards/rewards.js

```diff
@@ -77,15 +78,21 @@ function validateIntervalTimestamps(interval) {
 
 async function calculateOperatorsRewards(context, interval) {
   const { cache } = context
+
   const slaCalculator = await SLACalculator.initialize(context, interval)
+  const fraudDetector = await FraudDetector.initialize(context)
```

Comment:
Nitpick and definitely non-blocking: given that we are checking fraud before SLA and we put `isFraudulent` before `operatorSLA` in `OperatorRewards`, can we initialize `fraudDetector` before `slaCalculator`?